CHANGELOG.rst

```diff
@@ -2,30 +2,40 @@
 Changelog
 *********
 
-0.1.0 -- 2019-02-05
-===================
-* Initial public release
+1.0.1 -- 2019-09-27
+=================== 
+* Modified Doxygen config file to generate dependency graphs #376
```

Comment:
It would be nice to have the numbers link to the issue/PR in question.

Reply:
fixed.